src/transport/fusedev/linux_session.rs

```diff
@@ -89,7 +89,7 @@ impl FuseSession {
     }
 
     /// Expose the associated FUSE session file.
-    pub fn get_fuse_file(&mut self) -> Option<&File> {
+    pub fn get_fuse_file(&self) -> Option<&File> {
```

Comment:
There's another instance in `src/transport/fusedev/macos_session.rs`. Could you modify both of them? Thanks!